docs/core/building/index.md

```diff
@@ -0,0 +1,101 @@
+
+ * Debug (default)- This compiles the runtime with additional runtime checks (asserts).  These checks slow 
+   runtime execution but are really valuable for debugging, and is recommended for normal development and testing.  
+ * Release - This compiles without any development time runtime checks.  This is what end users will use but 
+   can be difficult to debug.   Passing 'release' to the build script select this.  
+
+In addition, by default the build will not only create the runtime executables, but it will also 
+build all the tests.   There are quite a few tests so this does take a significant amount of time
+that is not necessary if you want to experiment with changes.
+You can submit the building of the tests with the 'skiptests' argument to the build script.
+
+In order to get a build as quickly as possible type the following (using `\` as the directory separator, use `/` on Unix machines)
+```bat
+    .\build skiptests 
+```
+which will build the Debug flavor which has development time checks (asserts), or 
+```bat 
+    .\build release skiptests
+```
+to build the release (full speed) flavor.  You can find more build options with build by using the -? or -help qualifier.   
+
+#### Using Your Build
+
+The build places all of its generated files under the `bin` directory at the base of the repository.   There 
+is a `bin\Log` directory that contains log files generated during the build (Most useful when the build fails).
+The actual output is placed in a `bin\Product\[platform].[CPU architecture].[build type]` directory, such as `bin\Product\Windows_NT.x64.Release`.
+
+While the 'raw' output of the build is sometimes useful, normally you are only interested in the Nuget packages, which are placed in the `.nuget\pkg` subdirectory of the above output directory.
+
+There are two basic techniques for using your new runtime.
+
+ 1. **Use dotnet.exe and Nuget to compose an application**.
+    See [Using Your Build](https://github.com/dotnet/coreclr/blob/master/Documentation/workflow/UsingYourBuild.md) for instructions on creating a program that uses your new runtime by using the NuGet packages you just created and the'dotnet' command line interface.  This is the expected way non-runtime developers are likely to consume your new runtime.    
+
+ 2. **Use corerun.exe to run an application using unpackaged Dlls**.
+    This repository also defines a simple host called corerun.exe that does NOT take any dependency on NuGet.
+    This host has to be told where to get the necessary DLLs you actually use, and you have to manually gather them together.
+    This is the technique that all the tests in the repo use, and is useful for quick local 'edit-compile-debug' loop (e.g. preliminary unit testing).
+    See [Executing .NET Core Apps with CoreRun.exe](https://github.com/dotnet/coreclr/blob/master/Documentation/workflow/UsingCoreRun.md) for details on using this technique.
+
+### Building the CLI
+
+## Prerequisites
```

Comment:
This heading has the wrong level, it should be `####`, not `##`.

Also, the actual build instructions don't have their own heading, so it looks like they are part of Prerequisites, which is wrong.